### ISSUE TITLE
Tile for Flashy?

### ISSUE THREAD

**Issue by sphh** [Crazy-Marvin/Flashy#10]
Would it be possible to add a tile to quickly start `Flashy`? Similar to the built in, but inferior, flash light?

**Comment by CrazyMarvin** [comment]
What do you mean with tile? 
The icons that are shown above the notifications (like WiFi, flight mode, Bluetooth, GPS, etc.)? Or a widget for the Home screen?
Both would be useful but I don't know how to put Flashy up to the tiles above the notifications.

**Comment by sphh** [comment]
Yes, I mean those icons that are shown above the notifications when swiping down from the top.

I checked on my phone and these open source projects add tiles:

- AN2Linux: https://github.com/rootkiwi/an2linuxclient
- CPU Info: https://github.com/kamgurgul/cpu-info
- Open Camera: https://sourceforge.net/p/opencamera/code
- NetGuard: https://github.com/M66B/NetGuard

Maybe they can help?

**Comment by CrazyMarvin** [comment]
Can you please prepare some screenshots of their tiles so I can have a look, @sphh?

**Comment by sphh** [comment]
Here you can see the tiles for `NetGuard`, OpenCamera (`Camera`), AN2Linux and CPU Info (`Avg 273MHz`):
![Screenshot_20200729-153721](https://user-images.githubusercontent.com/4856462/88822084-fac7bf80-d1b2-11ea-9934-1831ba6c5381.png)

**Comment by CrazyMarvin** [comment]
Added in #52
